package.json

```diff
@@ -14,12 +14,20 @@
   "main": "bundles/angular-cesium.umd.js",
   "typings": "src/main.d.ts",
   "scripts": {
-    "cleanup": "rimraf src/*.ngfactory.ts src/*.ngstyle.ts src/*.ngsummary.json build bundles",
+    "clean_src": "rimraf src/*.d.ts src/*.metadata.json src/*.js src/*.js.map src/*.ngfactory.ts src/*.ngstyle.ts src/*.ngsummary.json",
+    "clean_components_src": "rimraf src/components/**/*.d.ts src/components/**/*.metadata.json src/components/**/*.js src/components/**/*.js.map src/components/**/*.ngfactory.ts src/components/**/*.ngstyle.ts src/components/**/*.ngsummary.json",
+    "clean_models_src": "rimraf src/models/**/*.d.ts src/models/**/*.metadata.json src/models/**/*.js src/models/**/*.js.map src/models/**/*.ngfactory.ts src/models/**/*.ngstyle.ts src/models/**/*.ngsummary.json",
+    "clean_pipes_src": "rimraf src/pipes/**/*.d.ts src/pipes/**/*.metadata.json src/pipes/**/*.js src/pipes/**/*.js.map src/pipes/**/*.ngfactory.ts src/pipes/**/*.ngstyle.ts src/pipes/**/*.ngsummary.json",
+    "clean_services_src": "rimraf src/services/**/*.d.ts src/services/**/*.metadata.json src/services/**/*.js src/services/**/*.js.map src/services/**/*.ngfactory.ts src/services/**/*.ngstyle.ts src/services/**/*.ngsummary.json",
+    "clean_utils_src": "rimraf src/utils/**/*.d.ts src/utils/**/*.metadata.json src/utils/**/*.js src/utils/**/*.js.map src/utils/**/*.ngfactory.ts src/utils/**/*.ngstyle.ts src/utils/**/*.ngsummary.json",
+    "cleanup": "npm run clean_src && npm run clean_components_src && npm run clean_models_src && npm run clean_pipes_src && npm run clean_services_src && npm run clean_utils_src && rimraf index.js index.js.map index.d.ts index.metadata.json build bundles",
     "start": "webpack-dev-server --inline --progress --port 8080",
     "lint": "tslint -c tslint.json --project tsconfig.aot.json",
     "test": "karma start",
     "ngc": "ngc -p tsconfig.aot.json",
     "aot": "webpack --config webpack.aot.config.js -p",
+    "rollup": "rollup -cza",
+    "rollup-bundle": "npm run cleanup && npm run lint && npm run ngc && npm run rollup",
     "bundle": "npm run cleanup && npm run lint && npm run ngc && npm run aot",
```

Comment:
1. should delete the webpack bundling 
2. what about the demo ? maybe make it work with angular-cli instead of webpack?